components/prism-hoon.js

```diff
@@ -0,0 +1,21 @@
+Prism.languages.hoon = {
+	'constant': /%(?:\.[ny]|[\w-]+)/,
+	'comment': {
+		pattern: /::.*/,
+		lookahead: true,
```

Comment:
Sorry, I missed this one @matildepark.

We don't have something called `lookahead`, we only have `lookbehind`. Please remove the `lookahead: true` here and in `class-name`. It doesn't do anything.